system/modules/core/elements/ContentDownloads.php

```diff
@@ -146,7 +146,7 @@ protected function compile()
 					$arrMeta['title'] = specialchars($objFile->basename);
 				}
 
-				$strHref = \Environment::get('request');
+				$strHref = $objPage->getFrontendUrl() . (Environment::get('queryString') ? '?' . Environment::get('queryString') : '');
```

Comment:
This is because the PR did not contain a change log entry. Also, there is not generally a change log entry for every commit, because that's what the commit history is for.